mlflow/tracking/fluent.py

```diff
@@ -1139,6 +1139,57 @@ def log_input(
     MlflowClient().log_inputs(run_id=run_id, datasets=datasets, models=[model])
 
 
+def log_inputs(
+    datasets: Optional[List[Dataset]] = None,
+    context: Optional[str] = None,
+    tags: Optional[Dict[str, str]] = None,
```

Comment:
Can we make this `List[DatasetInput]` instead so that it contains all of the relevant tags and introduce an easy way to convert a dataset into a DatasetInput? This new method is P1, so feel free to revisit this PR later, as long as we track it on the backlog :)

Reply:
Sounds good. I'll revisit this once P0 items are done.